lib/gs-plugin-loader.h

```diff
@@ -109,4 +109,6 @@ void            gs_plugin_loader_set_max_parallel_ops  (GsPluginLoader *plugin_l
 const gchar	*gs_plugin_loader_get_locale		(GsPluginLoader *plugin_loader);
 GPtrArray	*gs_plugin_loader_dup_copy_dests	(GsPluginLoader *plugin_loader);
 
+gboolean	gs_plugin_loader_app_copying (GsPluginLoader *plugin_loader, GsApp *app);
```

Comment:
Alignment is still off here, the `gs_` needs to align with the ones above, as does the `(Gs`. And `GsApp`... should be on the next line.

Reply:
(This is still true with the latest version, ef75c8f)

Reply:
Looks good now